r2/r2/templates/redditheader.compact

```diff
@@ -111,6 +111,11 @@
     <div class="menuitem">
       ${plain_link("search", "/search")}
     </div>
+    %if not isinstance(c.site, DefaultSR) and not c.cname:
```

Comment:
Why the `c.cname` check?

`not isinstance(c.site, DefaultSR)` will still create the link on multireddits and other subreddits that don't have sidebars.